MultiWorldServer/Server.cs

```diff
@@ -473,19 +476,13 @@ private void HandleReadyMessage(Client sender, MWReadyMessage message)
             sender.Room = message.Room;
             lock (_clientLock)
             {
-                if (unsavedResults.ContainsKey(sender.Room) && !unsavedResults[sender.Room].ContainsKey(sender.Nickname))
-                {
-                    // Room has started & still open to make sure nobody loses their save & sender wasn't in room originally
-                    SendMessage(new MWNumReadyMessage { Ready = -1, Names = "Could not ready, please change room name" }, sender);
-                    return;
-                }
-
                 if (!ready.ContainsKey(sender.Room))
                 {
-                    ready[sender.Room] = new Dictionary<ulong, RandoSettings>();
-                } 
-                
-                ready[sender.Room][sender.UID] = message.Settings;
+                    ready[sender.Room] = new Dictionary<ulong, (int, RandoSettings)>();
+                }
+
+                int readyId = (new Random()).Next();

```

Comment:
Low chances, but this isn't necessarily unique 🌵

Reply:
I think it's fine, i'm willing to take 1/2^32 odds of something going weird

Reply:
lol np, more than a few things need to go wrong for this to act as an issue